### DISCUSSION TITLE
Preferred citation for kart?

### DISCUSSION THREAD

**Discussion by alpha-beta-soup** [koordinates/kart#913] [Q&A] [answered]
I'm in the process of writing a data paper that (in part) leverages Kart for managing representations of data over time. How should I cite Kart?

**Comment by rcoup** [comment, marked as answer]
Hi,

I've added a [PR with some basic citation information](https://github.com/koordinates/kart/pull/914/files#), and I've setup Zenodo archiving so there'll be a DOI with our next release. Any feedback/suggestions welcome though 😃 

https://github.com/koordinates/kart/blob/facc5924dd1959a758676a0bfbad1d1756f9af89/CITATION#L1-L13
